### ISSUE TITLE
gn 3.4 - moment is not defined

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2020]
`/geonetwork/srv/fre/catalog.search?debug` does not load, error is
```
angular.js:68 Uncaught Error: [$injector:modulerr] Failed to instantiate module gn_search_georchestra due to:
Error: [$injector:modulerr] Failed to instantiate module gn_search_default due to:
Error: [$injector:modulerr] Failed to instantiate module gn_search due to:
Error: [$injector:modulerr] Failed to instantiate module gn_module due to:
Error: [$injector:modulerr] Failed to instantiate module gn due to:
Error: [$injector:modulerr] Failed to instantiate module gn_locale due to:
ReferenceError: moment is not defined
```

Indeed `/geonetwork/catalog/lib/moment+langs.min.js` is HTTP 404

**Comment by fvanderbiest** [comment]
Yet it is provided by the webapp: /var/lib/jetty/webapps/geonetwork/catalog/lib/moment+langs.min.js

**Comment by fvanderbiest** [comment]
The same curl query from the SP yields the 404 => the issue is probably related to the SP handling the `+` char.

**Comment by fvanderbiest** [comment]
Duplicate

**Comment by pmauduit** [comment]
URLs are messed up in the SP here:
https://github.com/georchestra/georchestra/blob/master/security-proxy/src/main/java/org/georchestra/security/Proxy.java?utf8=%E2%9C%93#L463

I'd like to retest under tomcat to check if this is still needed, IMHO we should stay the closest the request arrives on the SP, and don't try to rewrite things like this ...

**Comment by pmauduit** [comment]
It is worth to try removing the URLDecode() call in the method above, but we should pay attention not to introduce a regression. digging the git history, it has been introduced to ensure compatibility with tomcat, to be tested thoroughly tomorrow (ping @fvanderbiest).

**Comment by pmauduit** [comment]
sandbox env ready for runtime testing (tomcat)

**Comment by fvanderbiest** [comment]
Tried playing with it & it looks good.

**Comment by pmauduit** [comment]
Have you also tried GS (adding a workspace / layer or so) ? I guess that the piece of code I removed was related to GS compatibility.

**Comment by fvanderbiest** [comment]
Yes I did. Could not test GN3 because it's not functional on sdi-stable

**Comment by pmauduit** [comment]
> Could not test GN3 because it's not functional on sdi-stable

sandbox.georchestra.org or sdi-stable.georchestra.org ?

**Comment by fvanderbiest** [comment]
This one is indeed fixed, but now /geonetwork/srv/api/0.1/standards/iso19139/codelists/gmd%3AMD_TopicCategoryCode and several others end up as 404:

![capture du 2018-07-04 13-18-10](https://user-images.githubusercontent.com/265319/42274371-b702df20-7f8c-11e8-9956-11cf4b78fe66.png)

They load correctly before SP traversal:
![capture du 2018-07-04 13-19-33](https://user-images.githubusercontent.com/265319/42274431-f5dedb86-7f8c-11e8-8c97-9708ee1ec9cd.png)

**Comment by pmauduit** [comment]
Yet another fix, which would require thorough testing on the go... see https://github.com/georchestra/georchestra/pull/2118
